### ISSUE TITLE
Bundle import paths are invalid when packaged using pkg

### ISSUE THREAD

**Issue by DenizUgur** [SoftwareBrothers/adminjs#1229]
**Describe the bug**
I'm not entirely sure but this line might be the culprit on file [generate-user-component-entry.ts](https://github.com/SoftwareBrothers/adminjs/blob/1d0eee21a342098a27a953114d571e98943685bf/src/backend/bundler/generate-user-component-entry.ts#L29). Basically, this path generated by the bundler leads nowhere so using absolute path packaged might be the solution here.

<details>
  <summary>tsconfig.json</summary>
  

  ```json
{
	"compilerOptions": {
		"target": "ESNext",
		"module": "Node16",
		"jsx": "preserve",
		"lib": ["dom", "ESNext"],
		"allowJs": true,
		"outDir": "build",
		"rootDir": "src",
		"strict": true,
		"noImplicitAny": true,
		"esModuleInterop": true,
		"resolveJsonModule": true,
		"skipLibCheck": true,
		"forceConsistentCasingInFileNames": true,
		"sourceMap": true,
		"inlineSources": true
	}
}

  ```
</details>

<details>
  <summary>Error output</summary>

  ```js
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:198
        base = Object.assign(new Error(base.message), base);
                             ^

Error: 'default' is not exported by ../../../../../../../../../snapshot/webapp/server/build/admin/containers/bulkUser.jsx, imported by .adminjs/.entry.js
    at error (/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:198:30)
    at Module.error (/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:12560:16)
    at Module.traceVariable (/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:12919:29)
    at ModuleScope.findVariable (/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:11571:39)
    at Identifier.bind (/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:7570:40)
    at AssignmentExpression.bind (/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:5400:23)
    at ExpressionStatement.bind (/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:5400:23)
    at Program.bind (/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:5396:73)
    at Module.bindReferences (/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:12556:18)
    at Graph.sortModules (/snapshot/webapp/server/node_modules/rollup/dist/shared/rollup.js:23155:20) {
  code: 'MISSING_EXPORT',
  url: 'https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module',
  id: '/Users/deniz/Documents/PROJECTS/WEB PROJECTS/bus-fs/webapp/server/dist/.adminjs/.entry.js',
  pos: 36,
  loc: {
    column: 7,
    file: '/Users/deniz/Documents/PROJECTS/WEB PROJECTS/bus-fs/webapp/server/dist/.adminjs/.entry.js',
    line: 2
  },
  frame: '1: AdminJS.UserComponents = {}\n' +
    "2: import Component1 from '../../../../../../../../../../snapshot/webapp/server/build/admin/containers/bulkUser'\n" +
    '          ^\n' +
    '3: AdminJS.UserComponents.Component1 = Component1\n' +
    "4: import Component2 from '../../../../../../../../../../snapshot/webapp/server/build/admin/containers/bulkUser'",
  watchFiles: [
    '/Users/deniz/Documents/PROJECTS/WEB PROJECTS/bus-fs/webapp/server/dist/.adminjs/.entry.js',
    '/snapshot/webapp/server/build/admin/containers/fileUpload.jsx',
    '/snapshot/webapp/server/build/admin/containers/documentation.jsx',
    '/snapshot/webapp/server/build/admin/containers/bulkUser.jsx',
    '\x00rollupPluginBabelHelpers.js'
  ]
}
  ```
</details>

**Installed libraries and their versions**

- "@adminjs/express": "^5.0.0"
- "@adminjs/sequelize": "^3.0.0"
- "adminjs": "^6.1.4"
- "pkg": "^5.8.0"

**To Reproduce**
I expect this issue to be reproducible by just packaging any adminjs project. You just have to add compiled [tj]sx files as assets. If it doesn't do that I can try to create a minimal repository.

**Expected behavior**
It should start as normal when packaged. I'm using pkg to create optimized and minimal version of my server before deploying it to AWS so backward compatibility etc. is not important.

**Comment by dziraf** [comment]
Can you share a repository where I can reproduce this and some information about your OS etc? I've just tried reproducing in a project I'm working on privately and it passes although I'm getting a warning:
```
prebuild-install WARN install No prebuilt binaries found (target=v16.16.0 runtime=node arch=arm64 libc= platform=darwin)
```

**Comment by DenizUgur** [comment]
Hello, after investigating the problem I've found the root cause. Error log was already pointing out but the problem was my tsx files were being compiled like this:

```js
function Board() { ... }
exports.default = Board
````

When I switched to jsx and used the original export like this:

```js
export default Board() { ... }
```

The error was gone. So this is either a typescript related configuration problem or something about rollup configuration.
